### PR TITLE
feat: Added Avatars in users list and usersAutoComplete

### LINKED ISSUE #8869: Show user avatar in User Autocomplete and User list page
**Is your feature request related to a problem? Please describe.**
Currently the [`UserAutocompleteFormField`](https://github.com/ohcnetwork/care_fe/blob/develop/src/Components/Common/UserAutocompleteFormField.tsx#L82) and the Users list page does not show the avatar of the user.

**Describe the solution you'd like**
- [ ] Show the avatar of the user in each of the options for [`UserAutocompleteFormField`](https://github.com/ohcnetwork/care_fe/blob/develop/src/Components/Common/UserAutocompleteFormField.tsx#L82)
- [x] Show the avatar of the users in the Users list page ([here](https://github.com/ohcnetwork/care_fe/blob/develop/src/Components/Users/ManageUsers.tsx#L192))

**Additional context**
Image

Image

### PR DESCRIPTION
**Pull request by noufalrahim**
## Proposed Changes

- Fixes #8869 
- Added user avatar in users list page
- Added user avatar in userAutoComplete 
- Added profile_pic to userBareMinimum (optional property ['?'])

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Common/UserAutocompleteFormField.tsx [hunk 1]
```diff
@@ -148,11 +149,21 @@ export const LinkedFacilityUsers = (props: LinkedFacilitySearchProps) => {
...
-const userOnlineDot = (user: UserBareMinimum) => (
-  <div
-    className={classNames(
-      "mr-4 size-2.5 rounded-full",
-      isUserOnline(user) ? "bg-primary-500" : "bg-secondary-400",
-    )}
-  />
-);
+const userOnlineDot = (user: UserBareMinimum) => {
+  console.log(user);
```
Comment (rithviknishad) [inline src/Components/Common/UserAutocompleteFormField.tsx:153]:
```suggestion
```

File: src/Components/Common/UserAutocompleteFormField.tsx [hunk 2]
```diff
@@ -148,11 +149,21 @@ export const LinkedFacilityUsers = (props: LinkedFacilitySearchProps) => {
...
+    <div className="mr-2 flex items-center">
+      <div
+        className={classNames(
+          "mr-4 size-2.5 rounded-full",
+          isUserOnline(user) ? "bg-primary-500" : "bg-secondary-400",
+        )}
+      />
+      <Avatar
+        name={formatName(user)}
+        imageUrl={user.profile_picture}
```
Comment (rithviknishad) [inline src/Components/Common/UserAutocompleteFormField.tsx:164]: `profile_picture`? Are you sure this is correct?

It's not working. The user shown below has a profile picture uploaded, but not shown here. Please test your PR :) 

image

File: src/Components/Users/models.tsx [hunk 3]
```diff
@@ -21,6 +21,7 @@ export type UserBareMinimum = {
   email: string;
   user_type: UserRole;
   last_login: string | undefined;
+  profile_picture?: string;
```
Comment (rithviknishad) [inline src/Components/Users/models.tsx:24]:
```suggestion
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Comment by rithviknishad** [comment]
@noufalrahim Do follow the PR checklist. Screenshots / videos on PR body would really help.

File: src/Components/Common/UserAutocompleteFormField.tsx (hunk 2 above)
Comment (noufalrahim) [reply src/Components/Common/UserAutocompleteFormField.tsx:164]: How to get Rithvik Nishad as a suggestion? When typed it is showing no option found

Comment (rithviknishad) [reply src/Components/Common/UserAutocompleteFormField.tsx:164]: You can try logging in to the doctor credentials provided in the README and upload a profile pic for that user, and try to create a consultation for any patient.

**Pull request closed without merging**